nGraph: Fix TopK output sorting by index (#3092)
openvinotoolkit/openvino: docs/ops/sort/TopK_3.md

> 

Reviewer: Please, add a "Detailed description section" where all combinations of values for "mode" and "sort" will be explicitly described (like it is done in the ticket).

* *axis*

>   * **Description**: Specifies the axis along which to make the selection.

Reviewer 2: ```suggestion
  * **Description**: Specifies the axis along which the values are retrieved.
```
